### ISSUE TITLE
As a content editor, I want to merge item records so that I can combine item records and associate borrowing events when I've identified duplicated items.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/mep-django#107]
## Notes for development
- should work similar to person merge does: select items from list view and choose a merge action
- Borrowing events should be combined on the main record
- Preserve creator information; warn if creator information is present and does not match (but ok if not present / only present on one item); warn if publication date is set for both but don’t match; should warn but allow to merge
- User should pick primary record; default to the one with the most information
- Copy publication date, uri; combine notes
- Add note: merged with db id on date
- return to item list page where the selection was triggered after completion (same as person merge)

**Comment by rlskoeser** [comment]
Blocked pending OCLC reconciliation work and decisions & data migration for work/edition data modeling.

**Comment by rlskoeser** [comment]
This is no longer blocked by OCLC work, but it's not clear to me if we still need the functionality or if remaining book work can be done without a merge feature.

**Comment by rlskoeser** [comment]
book merge feature no longer needed
